docs/whatsnew/0.6.0.rst

```diff
@@ -0,0 +1,2 @@
+.. changelog::
+   :towncrier:
```

Comment:
This should fix it, you need this to be a path to your repo root (where pyproject.toml) is.
```suggestion
   :towncrier: ../../
```

I will explicitly catch this error in the plugin.

Reply:
https://github.com/OpenAstronomy/sphinx-changelog/pull/5